README.md

```diff
@@ -58,14 +58,14 @@ If you'd like to help the project by supporting us financially we accept and gre
 
 ## Source Code
 
-Getting started with Intersect couldn't be easier. Make sure you have Visual Studio 2017 Community (or later) installed. 
+Getting started with Intersect couldn't be easier. Make sure you have Visual Studio 2019 Community installed. 
```

Comment:
Should this really be in this pr?

Reply:
This was previously approved and merged into the main branch by @lodicolo . I don't see why this shouldn't be in the Dev branch for now either, plus to this, the authors have been updated as well, since @blinkuz helped along with thir PR.

Reply:
Should be its own pr though,. entirely unrelated to this feature addition.

Reply:
👍🏽 Reversed this commit from this PR, 
and opened a new PR with this change, as suggested by the contributor @Cheshire92   - >  #585